src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupOperation/BackupOperation.cs

```diff
@@ -211,12 +213,12 @@ public override void Execute()
                 {
                     string destName = Convert.ToString(this.backupInfo.BackupPathList[i], System.Globalization.CultureInfo.InvariantCulture);
                     int deviceType = (int)(this.backupInfo.BackupPathDevices[destName]);
+                    int backupDeviceType =
```

Comment:
We're parsing the backupDeviceType here but this doesn't seem to be the one that's used in the below code - am I missing something here?